planet/System.c4g/LanguageDE.txt

```diff
@@ -1,10 +1,10 @@
 IDS_LANG_NAME=Deutsch
-IDS_LANG_INFO=Original-Sprachpaket von RedWolf Design.
```

Comment:
I'm not sure how we should word this. With your changes it is not only extended, but also modified. Same for english of course.

Reply:
I also thought about it but there are copyrights on the strings by the authors.